config.py

```diff
@@ -26,8 +28,14 @@ def __init__(self, name, parser, globals):
         '''
         from domain import Domain
 
+        self.contact = parser.get(name, 'Contact', fallback=None)
+        self.hardware_model = parser.get(name, 'Hardware-Model', fallback=None)
+        self.hostname = parser.get(name, 'Hostname', fallback=socket.gethostname())
         self.name = name
         self.interfaces = list(map(str.strip, parser.get(name, 'Interfaces', fallback='').split(',')))
+        self.is_gateway = (parser.get(name, 'VPN', fallback='True')).lower() in ['true', 'yes']
+        self.longitude = float(parser.get(name, 'Longitude', fallback=None))
```

Comment:
The fload conversion has to be done differently.

Reply:
Conversion Errors are now caught: https://github.com/ffnord/mesh-announce/pull/63#issuecomment-778792117